Detectors/MUON/MCH/Base/include/MCHBase/Digit.h

```diff
@@ -28,14 +28,32 @@ namespace mch
 class Digit
 {
  public:
+  struct Time {
+    union {
+      // default value
+      uint64_t time = 0x0000000000000000;
+      struct {                       ///
+        uint32_t sampaTime : 10;     /// bit 0 to 9: sampa time
+        uint32_t bunchCrossing : 20; /// bit 10 to 29: bunch crossing counter
+        uint32_t reserved : 2;       /// bit 30 to 31: reserved
+        uint32_t orbit;              /// bit 32 to 63: orbit
+      };                             ///
+    };
+    uint64_t getBXTime()
+    {
+      uint64_t result = bunchCrossing;
```

Comment:
``result`` is assigned and not used? I guess you wanted to have ``return result + (sampaTime * 4);``?

Reply:
You are correct. Fixed.